mantis-control-plane/mantis-control-plane-server/src/main/java/io/mantisrx/master/resourcecluster/ResourceClusterGatewayAkkaImpl.java

```diff
@@ -53,7 +53,7 @@ class ResourceClusterGatewayAkkaImpl implements ResourceClusterGateway {
 
     private <In, Out> Function<In, CompletableFuture<Out>> withThrottle(Function<In, CompletableFuture<Out>> func) {
         return in -> {
-            if (rateLimiter.tryAcquire(1, TimeUnit.SECONDS)) {
+            if (rateLimiter.tryAcquire(200, TimeUnit.MILLISECONDS)) {
```

Comment:
Ideally, for these things, it would be nice if we could just configure them at the configuration level.